content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -36,12 +36,13 @@ It is important to preserve label mapping, so that Kiali can filter on _app_ and
         regex: __meta_kubernetes_pod_label_(.+)
 ```
 
-Additionally, you have to tell Kiali the URL of that Prometheus server: either by setting the environment variable `PROMETHEUS_CUSTOM_METRICS_URL` or by modifying the Kiali _ConfigMap_:
+Additionally, you have to tell Kiali the URL of that Prometheus server by setting the proper configuration setting in the Kiali CR:
```

Comment:
Wordy but not a blocker.  maybe:

Additionally, you must configure the Prometheus server URL in the Kiali CR:

Reply:
fine.